int32 equipletID
int32 moduleID
int32 newState